ERCS/erc-7540.md

```diff
@@ -354,7 +354,7 @@ MUST be emitted when a deposit Request is submitted using the `requestDeposit` m
       indexed: true
       type: uint256
     - name: sender
-      indexed: false
+      indexed: true
```

Comment:
This is unfortunately not possible because there cannot be more than 3 indexed parameters